Skype/SDN/articles/devicerendernotfunctioningeventratio-element-qualitypropertiestype-complextype-skype-sdn-2-2-c.md

```diff
@@ -1,5 +1,6 @@
 ﻿---
 title: DeviceRenderNotFunctioningEventRatio element (QualityPropertiesType complexType) Schema C
+description: Describes the Schema C iteration of the DeviceRenderNotFunctioningEventRatio element and provides the element's definition and element information.
```

Comment:
Describes the Schema C iteration of the DeviceRenderNotFunctioningEventRatio element and provides the element's definition and information.